plugins/pencilblue/controllers/actions/admin/sites/localization.js

```diff
@@ -0,0 +1,268 @@
+/*
+ Copyright (C) 2015  PencilBlue, LLC
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+var path = require('path');
+var fs = require('fs');
+
+module.exports = function(pb) {
+
+    //pb dependencies
+    var util = pb.util;
+
+    /**
+     * Saves the site's Localization settings
+     */
+    function Localization(){}
+    util.inherits(Localization, pb.BaseAdminController);
```

Comment:
There is already a localization API controller (controllers/api/localization_controller.js).  All API based localization interactions should reside there.